package.json

```diff
@@ -260,6 +260,7 @@
     "dot-prop": "6.0.1",
     "hosted-git-info": "4.0.2",
     "immer": "9.0.6",
+    "minimist": "npm:minimist-lite@2.2.0",
```

Comment:
Fixes a vulnerability found in `minimist`: https://github.com/substack/minimist/issues/164#issuecomment-1074349919

Reply:
Could this go in resolutions rather than package 🤔

Reply:
This is under `resolutions`, or maybe I don't know what you mean?

Reply:
Yep, sorry though this was added as package. Nothing to see here 👀